### PR TITLE
FIX | use token from config

### PR DESCRIPTION
**Pull request by tomasznapieralski**
# Description
- use token from config

### CODE REVIEW

File: app/views/users/user_skill_rates/history.html.haml [hunk 1]
```diff
@@ -3,4 +3,4 @@
     %span Your skills history
     = link_to 'Skill rating', user_skill_rates_path, class: 'btn btn-xs btn-default'
 
-  = react_component('user_skill_history', {user_id: user.id, skill_categories: skill_categories})
+  = react_component('user_skill_history', {user_id: user.id, skill_categories: skill_categories, api_token: AppConfig.api_token})
```
Comment (netguru-hound) [inline app/views/users/user_skill_rates/history.html.haml]: Line is too long. [129/80]

File: app/views/users/user_skill_rates/history.html.haml [hunk 2]
```diff
@@ -3,4 +3,11 @@
     %span Your skills history
     = link_to 'Skill rating', user_skill_rates_path, class: 'btn btn-xs btn-default'
 
-  = react_component('user_skill_history', {user_id: user.id, skill_categories: skill_categories})
+  = react_component(                      |
```
Comment (netguru-hound) [inline app/views/users/user_skill_rates/history.html.haml]: Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.

File: app/react/components/skills/user_skill_history.jsx [hunk 3]
```diff
@@ -87,7 +87,7 @@ export default class UserSkillHistory extends Component {
       dataType: 'json',
       data: {
         category,
-        token: 'miamiamia',
```
Comment (bardzusny) [inline app/react/components/skills/user_skill_history.jsx]: :D

Comment (tomasznapieralski) [inline app/react/components/skills/user_skill_history.jsx]: well... :D

File: spec/controllers/api/v2/user_skill_rates_controller_spec.rb [hunk 4]
```diff
@@ -4,8 +4,8 @@
   describe 'GET #index' do
     let(:token) { AppConfig.api_token }
     let(:user) { create(:user, email: 'john.smith@netguru.pl') }
-    let(:skill1) { create(:skill, name: 'ruby') }
-    let(:skill2) { create(:skill, name: 'rails') }
+    let(:skill1) { create(:skill, name: 'killingspree') }
+    let(:skill2) { create(:skill, name: 'spree') }
```
Comment (bardzusny) [inline spec/controllers/api/v2/user_skill_rates_controller_spec.rb]: I don't mind, but some people are (understandably?) annoyed when stumbling upon joke data in tests. :>

Comment (bardzusny) [inline spec/controllers/api/v2/user_skill_rates_controller_spec.rb]: Yes :)

**Pull request merged**